pioinstaller/__main__.py

```diff
@@ -40,6 +41,17 @@ def pack(target):
     return packer.pack(target)
 
 
+@cli.group()
+def check():
+    pass
+
+
+@check.command()
+def python():
+    assert python_check()
+    click.echo("Python check was successful.")
```

Comment:
1) Add to util 
```
def get_pythonexe_path():
    return os.environ.get("PYTHONEXEPATH", normpath(sys.executable))
```
2. Return `click.echo("The Python `%s` interpreter is compatible." % get_pythonexe_path())`